mce-dbus.c

```diff
@@ -3831,78 +3831,65 @@ static gboolean config_set_dbus_cb(DBusMessage *const msg)
  * @return TRUE if message matches the rules,
  *	   FALSE if not
  */
-static gboolean check_rules(DBusMessage *const msg,
-			    const char *rules)
-{
-	if (rules == NULL)
-		return TRUE;
-	rules += strspn(rules, " ");;
```

Comment:
Btw, this seems a bit repeating thingie. 
$ git grep ";;"|grep -v for|wc -l
14